.dockerignore

```diff
@@ -0,0 +1,7 @@
+*
```

Comment:
What is this dockerignore for?

Reply:
The docker daemon only looks for `.dockerignore` in the root of the current build context. Since this context is the project root when building the lab / annotation server, we ~need to~ place it here so that the files that we don't require for the build process are ignored.

[For reference](https://medium.com/@LihauTan/took-me-hours-to-realise-why-docker-build-ignores-my-dockerignore-and-this-is-what-ive-learned-2f87c770ea9c).